### DISCUSSION TITLE
Input Select Question

### DISCUSSION THREAD

**Discussion by FrankJaco** [tronikos/androidtvremote2#10] [General]
I have 3 Google TV's and one embedded Android TV. This integration is great.  Have 99.9% of what I want to do working already. Thanks for your efforts!

One thing I cannot figure out is how to send the command to access the "Input Select" function? On the actual remotes the button looks like the mdi:import icon but is used to switch TV inputs.

**Comment by heikkikii** [comment]
same question here - arrived to this repo linked from Home Asisstant's new (2023.05) Android TV Remote integration.

For other keys sending the command like "VOLUME_DOWN" (without the quotes) works.

For input select I tried 
KEYCODE_TV_INPUT
TV_INPUT

thinking they may be right key - but didnt work :-(

**Comment by tronikos** [comment]
I haven't found a way either. I was hoping `TV_INPUT_HDMI_1` to work but on my device it only works if I'm on the input select menu. Maybe we can launch the input select app with a deep link? I haven't found time to investigate. If you want to explore that option you should read https://community.home-assistant.io/t/android-tv-remote-app-links-deep-linking-guide/567921 An alternative also mentioned in that guide is to use the "button mapper" app.

**Comment by briodan** [comment]
same issue here with a Google TV, the command executes in dev tools but no action on the TV, it also does not do anything if i open the input select screen.

**Comment by DeKi90** [comment]
What I found to work previously using the "old" Android TV integration in Home Assistant aka Android Debug Bridge, sending the command "INPUT".
This runs ```adb shell input keyevent INPUT```, which works like a charm. However, I cannot find any mention of what that actually maps to, nor do I have the means to figure out which app that calls.
